hw6/hw6.py

```diff
@@ -0,0 +1,18 @@
+def flatten(items):
+    for x in items:
+        if hasattr(x, '__iter__') and not isinstance(x, str):
+            yield from flatten(x)
+        else:
+            yield x
+
+def merge_elems(*elems):
```

Comment:
strings of length >1 should be processed as other iterable objects
example input:
a = [1, 2, 3]
b = 6
c = 'zhaba'
d = [[1, 2], [3, 4]]
actual output: 1 2 3 6 zhaba 1 2 3 4
expected: 1 2 3 6 z h a b a 1 2 3 4